requirements/extras.txt

```diff
@@ -3,3 +3,4 @@ gradio>=2.9.4
 tensorboard
 wandb==0.12.17
 GitPython
```

Comment:
Would it be an idea to remove this from Anomalibs requirements and add it as a prerequisite in the README of the notebooks instead?

Reply:
I think these were moved https://github.com/openvinotoolkit/anomalib/pull/870/files#diff-3d0f7ff059ea9ec68b83f0d1a6eee038151708acf96afe054eb44a9d92786f80

Reply:
Oh you mean GitPython. Yeah let's do that

Reply:
How about requirements/notebooks.txt?